packages/validator/src/rules/no-undefined-path-parameters/__test__/no-undefined-path-parameters.test.ts

```diff
@@ -0,0 +1,14 @@
+import path from "path";
+import { getViolationsForRule } from "../../../testing-utils/getViolationsForRule";
+import { NoUndefinedPathParametersRule } from "../no-undefined-path-parameters";
+
+describe("no-undefined-path-parameters", () => {
+    it("simple", async () => {
+        const violations = await getViolationsForRule({
+            rule: NoUndefinedPathParametersRule,
+            absolutePathToDefinition: path.join(__dirname, "fixtures", "simple"),
+        });
+
+        expect(violations).toMatchSnapshot();
```

Comment:
the function is `toMatchInlineSnapshot()` and then run the test, but that wasn't that helpful actually. You can just do `expect(violations).toEqual([])`, and run the test. You'll get an error that the test failed and it'll print the actual value, and then you can just copy and paste into the test:
```
// good to type the `actual` so we get compile breaks if we change what a rule violation looks like
const actual: RuleViolation[] = [ ... ]

expect(violations).toEqual(actual);
```

Reply:
Ah gotcha -- i tried that it was just annoying to get bolded and colored values copy + pasted